src/Sarif.Viewer.VisualStudio/ResultTextMarker.cs

```diff
@@ -17,8 +17,8 @@
 namespace Microsoft.Sarif.Viewer
 {
     /// <summary>
-    /// This class represents an instance of "highlighed" line in the editor, holds necessary Shell objects and logic 
-    /// to managed lifecycle and appearance.
+    /// This class represents an instance of "highlighted" line in the editor, holds necessary Shell objects and logic 
+    /// to managed life cycle and appearance.
     /// </summary>
     public class ResultTextMarker
```

Comment:
>ResultTextMarker [](start = 17, length = 16)

If this class represents an instance of a highlighted line, it should be called `HighlightedLine`. I'm a big fan of naming things what they are.

Reply:
Note to self: Does it represent a line, or a single highlight on a line (there might be more than one).

---
In reply to: [482124036](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r482124036) [](ancestors = 482124036)

Reply:
How about "LocationTextMarker" ?

---
In reply to: [482421238](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r482421238) [](ancestors = 482421238,482124036)

Reply:
Let's talk more about this. I'm beginning to think the name might be ok and the comment might be wrong. A SARIF region can cover a range of colums in a line; it doesn't have to cover a whole line. So if this class represents a highlight for a single region, rather than always representing a whole line, then the comment should be "... an instance of a 'highlighted' region in the  editor", and the class name might be `RegionTextMarker`.

---
In reply to: [482445581](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r482445581) [](ancestors = 482445581,482421238,482124036)